src/ui/hexedit.cc

```diff
@@ -434,6 +435,7 @@ void HexEdit::paintEvent(QPaintEvent *event) {
   painter.drawText(startMargin_ - startPosX_,
                    separatorLength - horizontalAreaSpaceWidth_,
                    statusBarText());
+#endif
 
   for (auto rowNum = startRow_;
```

Comment:
After removing the code above, we now have an quite big empty area at bottom of our hex-view - maybe we should do something about it ?